src/components/RegistrationForm.js

```diff
@@ -0,0 +1,97 @@
+import React, { Component } from 'react'
+import HomeNav from './HomeNav'
+import { Router, Route, Link } from 'react-router'
+
+
+export default class RegistrationForm extends Component {
+  constructor( props) {
+    super ( props )
+    this.state = {fullName: '', email: '', password: '', phoneNumber: '', company: ''}
+
+  }
+
+  componentDidMount() {
+    const postIsHappening = {
+      method: 'POST', mode: 'cors', headers: new Headers({
+      'Content-Type': 'application/json',
+      'Accept': 'application/json'
+      })
+    }
+    const postString = `http:localhost:5000/api/v1/users`
+    fetch( postString, postIsHappening )
+    .then( data => data.json())
+  }
+
+  handleName( event ) {
```

Comment:
Dry out these handlers by making one handler that takes state key and event as arguments>`handleInputEvent = ( event, stateKey ) => { this.setState({ [stateKey]: event.target.stateKey })}`